### PR TITLE
Ice session definition

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

**Comment by rshpount** [comment]
Not sure this is correct. ICE session is something that continues until ufrag/passwords change.  The same candidates can be re-used across multiple ICE sessions.

**Comment by rshpount** [comment]
I think better definition will be:

An ICE session consists of all ICE-related actions starting with the candidate gathering, followed by the interactions (candidate exchange, connectivity checks, nominations and keep-alives) between the ICE agents until all the candidates are released or ICE-restart is initiated by ufrag/password change.

**Comment by cdh4u** [comment]
Roman's suggestion looks good.

File: draft-ietf-ice-rfc5245bis.xml [hunk 1]
```diff
@@ -644,6 +644,12 @@ This specification makes use of the following additional terminology:
 
 <t><list style="hanging">
 
+<t hangText="ICE Session:">
+An ICE session consists of all ICE-related actions starting with the candidate gathering,
+followed by the interactions (candidate exchange, connectivity checks,
+nominations and keep-alives) between the ICE agents until the candidates
+are released (due to an ICE restart or another reason). </t>
+
```
Comment (pthatcherg) [inline draft-ietf-ice-rfc5245bis.xml]: Why not just say an ICE session is all ICE activity tied to a candidate exchange with a specific ufrag/pwd?  Every time you do a candidate exchange with a new ufrag/pwd, you have a new ICE session.    I don't think we need to say all the rest.

**Review by pthatcherg — COMMENTED** [review]
Why not just say an ICE session is all ICE activity tied to a candidate exchange with a specific ufrag/pwd? Every time you do a candidate exchange with a new ufrag/pwd, you have a new ICE session. I don't think we need to say all the rest.

**Comment by akeranen** [comment]
I think the current proposed explanation (one with more details) is good. Would suggest merging this.

**Comment by cdh4u** [comment]
My suggestion is to merge, submit a new version, and then continue working on Peter's issues.

**Comment by stpeter** [comment]
👍

**Pull request merged**